client/blocks/comment-button/index.jsx

```diff
@@ -18,29 +18,46 @@ import { getPostTotalCommentsCount } from 'state/comments/selectors';
 
 class CommentButton extends Component {
 	static propTypes = {
-		onClick: PropTypes.func,
-		tagName: PropTypes.string,
 		commentCount: PropTypes.number,
+		link: PropTypes.string,
+		onClick: PropTypes.func,
 		showLabel: PropTypes.bool,
+		tagName: PropTypes.string,
+		target: PropTypes.string,
 	};
 
 	static defaultProps = {
-		onClick: noop,
-		tagName: 'li',
-		size: 24,
 		commentCount: 0,
+		link: null,
+		onClick: noop,
 		showLabel: true,
+		size: 24,
+		tagName: 'li',
+		target: null,
 	};
 
 	render() {
-		const { translate, commentCount, onClick, showLabel, tagName: containerTag } = this.props;
+		const {
+			commentCount,
+			link,
+			onClick,
+			showLabel,
+			tagName: containerTag,
+			target,
+			translate,
+		} = this.props;
 
 		return React.createElement(
 			containerTag,
-			{
-				className: 'comment-button',
-				onClick,
-			},
+			omitBy(
+				{
+					className: 'comment-button',
+					href: link,
```

Comment:
It's not uncommon in our components to use directly `href` as prop, and it would remove this rename.

Another thing: since `link` is only needed in case the tag is `a`, we could probably better protect it with a ternary like:
`href: 'a' === containerTag ? link : null,`

Reply:
Good catch! Nice, much cleaner 👍